docs/quickstart/create-and-publish-a-package-using-visual-studio-net-framework.md

```diff
@@ -1,162 +1,176 @@
 
-You run `nuget spec` only once to generate the initial manifest. When updating the package, you either change values in your project or edit the manifest directly.
+1. With your project open in **Solution Explorer**, open a command prompt by selecting **Tools** > **Command Line** >  **Developer Command Prompt**.
 
-1. Open a command prompt and navigate to the project folder containing `AppLogger.csproj` file.
+   The command prompt opens in your project directory where the `AppLogger.csproj` file is located.
 
-1. Run the following command: `nuget spec AppLogger.csproj`. By specifying a project, NuGet creates a manifest that matches the name of the project, in this case `AppLogger.nuspec`. It also include replacement tokens in the manifest.
+1. Run the following command: `nuget spec AppLogger.csproj`.
 
-1. Open `AppLogger.nuspec` in a text editor to examine its contents, which should appear as follows:
+   NuGet creates a manifest that matches the name of the project, in this case `AppLogger.nuspec`. It also includes replacement tokens in the manifest.
+
+1. Open `AppLogger.nuspec` in a text editor to examine its contents, which will be similar to the following code:
 
     ```xml
     <?xml version="1.0"?>
     <package >
       <metadata>
         <id>Package</id>
         <version>1.0.0</version>
-        <authors>YourUsername</authors>
-        <owners>YourUsername</owners>
+        <authors>Your username</authors>
+        <owners>Your username</owners>
         <license type="expression">MIT</license>
-        <projectUrl>http://PROJECT_URL_HERE_OR_DELETE_THIS_LINE</projectUrl>
-        <iconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
+        <projectUrl>http://ENTER_PROJECT_URL_HERE_OR_DELETE_THIS_LINE</projectUrl>
+        <iconUrl>http://ENTER_ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Package description</description>
         <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
-        <copyright>Copyright 2019</copyright>
+        <copyright>Copyright 2022</copyright>
         <tags>Tag1 Tag2</tags>
       </metadata>
     </package>
     ```
 
 ## Edit the manifest
 
-1. NuGet produces an error if you try to create a package with default values in your `.nuspec` file, so you must edit the following fields before proceeding. See [.nuspec file reference - optional metadata elements](../reference/nuspec.md#optional-metadata-elements) for a description of how these are used.
+1. Edit the following properties before proceeding. Otherwise, if you try to create a NuGet package with the default values in your `.nuspec` file, an error occurs. For information about these properties, see [Optional metadata elements](../reference/nuspec.md#optional-metadata-elements):
 
     - licenseUrl
     - projectUrl
     - iconUrl
```

Comment:
best to check `nuget spec`, but I don't think that iconUrl is added to the default nuspec any longer.  I expect that if the customer does keep/use `iconUrl`, that pack should genreate a warning saying it's deprecated, and I think we should guide customers to experiences that do not output warnings.

Reply:
You're correct, iconUrl is deprecated in the .nuspec. reference. I'll remove it.